cordlib/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/CordaTestBase.kt

```diff
@@ -115,40 +113,50 @@ open class CordaTestBase {
 
     @Before
     fun commonSetup() {
-        TestConfigurationsProvider.provider = object : TestConfigurationsProvider {
-            override fun getConfig(name: String): Configuration? {
-                // Watch carefully for these hard-coded values
-                // Now we assume that issuer(indy trustee) is the first created node from SomeNodes
-                return if (name == "Trustee") {
-                    ConfigurationMap(
-                        mapOf(
-                            "indyuser.walletName" to name,
-                            "indyuser.role" to "trustee",
-                            "indyuser.did" to "V4SGRU86Z58d6TV7PBUe6f",
-                            "indyuser.seed" to "000000000000000000000000Trustee1",
-                            "indyuser.genesisFile" to PoolManager.TEST_GENESIS_FILE_PATH
-                        )
-                    )
-                } else ConfigurationMap(
-                    mapOf(
-                        "indyuser.walletName" to name + random.nextLong().absoluteValue,
-                        "indyuser.genesisFile" to PoolManager.TEST_GENESIS_FILE_PATH
-                    )
-                )
-            }
-        }
-
         net = InternalMockNetwork(
             cordappPackages = listOf("com.luxoft.blockchainlab.corda.hyperledger.indy"),
-            networkParameters = testNetworkParameters(maxTransactionSize = 10485760 * 5)
+            networkParameters = testNetworkParameters(maxTransactionSize = 10485760 * 5),
+            defaultFactory = CordaTestBase::MockIndyNode
         )
     }
 
+    open class MockIndyNode(args: MockNodeArgs) : InternalMockNetwork.MockNode(args) {
+
+        companion object {
+            private val sessionId = Random().nextLong().absoluteValue.toString()
+            val TEST_GENESIS_FILE_PATH by lazy {
+                this::class.java.classLoader.getResource("docker_pool_transactions_genesis.txt").file
```

Comment:
why dont we use com.luxoft.blockchainlab.hyperledger.indy.utils.PoolManager#getTEST_GENESIS_FILE_PATH?